pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -56,6 +58,14 @@ const (
 	// Maximum allowed value of ofPortRequest.
 	ofPortRequestMax = 65279
 	hardwareOffload  = "hw-offload"
+
+	AntreaIFTypeKey = "antrea-type"
+	AntreaGateway   = "gateway"
+	AntreaContainer = "container"
+	AntreaTunnel    = "tunnel"
+	AntreaUplink    = "uplink"
+	AntreaHostIF    = "hostLink"
```

Comment:
We don't call ports belonging to container "containerLink", should it be called "host"?